### ISSUE TITLE
code issue 2

### ISSUE THREAD

**Issue by wgkwgk** [majumderb/compac#10]
**in your paper, there is a training setting like**

**Hyperparameters:** Following (Wolf et al., 2019)
we use history size 2 (i.e. 4 previous utterances).
We use AdamW optimizer (Loshchilov and Hutter,
2017) and the learning rate was set at 6.25e − 5
with a linear decay of step size 10−1 per epoch.
The baseline in REINFORCE was done with a discounted moving average with a ratio of 0.99. The
REINFORCE loss coefficient was set at 0.8 and the
language modeling loss coefficient was set to 1.0.

**Training:** Each model converged in 3 epochs on
an average with batch size 4 in a TITAN X (Pascal)
GPU that took 12 hours in total. While training,
we only observe perplexity on the validation set to
employ an early-stopping criteria.

however, why is that different with the command in train.py like

python3 -m models.reinforce_model.train --dataset_path=/data3/bodhi/data/personachat/weak_label_comet_personachat/personachat_self_original_comet_scores_alignlabels.expanded_persona_preprocessed.json --model_checkpoint=gpt2 --gradient_accumulation_steps=4 --lm_coef=2.0 --mc_coef=0.0 --max_history=2 --n_epochs=10 --num_candidates=1 --personality_permutations=1 --train_batch_size=2 --valid_batch_size=2 --do_train --training_type=reinforce --use_baseline --moving_avg_ratio=0.99 --reinforce_loss_coef=0.8 --lr=1e-4 --log_dir models/reinforce_model/ --exp_name EXP_NAME

especially about lm_coef, mc_coef, n_epochs, train_batch_size, lr

**Comment by majumderb** [comment]
Hi, please follow the hyperparameter settings provided in the paper. The command in the train.py is an example.
